### PR TITLE
DAOS-9333 test: Update SWIM-related env vars used in tests

### PR DESCRIPTION
**Pull request by shimizukko**
Instead of CRT_SWIM_RPC_TIMEOUT, use SWIM_PING_TIMEOUT.

volbigio tag is for bigio.py. Because the other change is made in server_utils_params.py, run the pr tests.

Skip-unit-tests: true
Test-tag: pr volbigio
Signed-off-by: Makito Kano <makito.kano@intel.com>

### CODE REVIEW

File: src/tests/ftest/daos_vol/bigio.py [hunk 1]
```diff
@@ -15,6 +16,7 @@ class DaosVol(VolTestBase):
     :avocado: recursive
     """
 
+    @skipForTicket("DAOS-6654")
```
Comment (daltonbohning) [inline src/tests/ftest/daos_vol/bigio.py:19]: Should we be skipping this? I'm curious if there is a reason why it wasn't skipped initially: https://daosio.atlassian.net/browse/DAOS-6654

**Review by daltonbohning — APPROVED** [review]

File: src/tests/ftest/daos_vol/bigio.py (hunk 1 above)
Comment (shimizukko) [reply src/tests/ftest/daos_vol/bigio.py:19]: I read the comments in the ticket, but not sure why it wasn't skipped initially either.

I reported this failure in DAOS-10198, but Scott said this is the known issue reported in DAOS-6654, so I added this tag.

Comment (daltonbohning) [reply src/tests/ftest/daos_vol/bigio.py:19]: In that case, when this PR lands, we should probably leave a comment on DAOS-6654 to unskip the test when fixed.

Comment (shimizukko) [reply src/tests/ftest/daos_vol/bigio.py:19]: I created a ticket to remove the tag. Thanks for the reminder.

**Review by dinghwah — APPROVED** [review]
Adding Maureen to review for the test skip for ticket DAOS-6654.

**Pull request merged**